### PR TITLE
Modified Color Swatch Fix

### PR DESCRIPTION
**Pull request by benblack75**
Signed-off-by: Benjamin Black <benblac@amazon.com>

Canceled previous PR- accidentally added files not related to this issue. Modified the files list to contain just files for color swatch fix.

### CODE REVIEW

**Comment by benblack75** [comment]
LYN-4719

File: Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material [hunk 1]
```diff
@@ -6,9 +6,9 @@
     "properties": {
         "baseColor": {
             "color": [
-                0.43414968252182007,
-                0.029556725174188614,
-                0.03955138474702835,
+                1.0,
+                0.0,
+                0.0038452735170722008,
```
Comment (invertednormal) [inline Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material:11]: What's the source of this? It doesn't seem to be in line with red from the macbeth colorchecker - see https://en.wikipedia.org/wiki/ColorChecker

**Review by invertednormal — CHANGES_REQUESTED** [review]

File: Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material (hunk 1 above)
Comment (Adi-Amazon) [reply Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material:11]: +1 on that.
The old existing was wrong but so is this.

Comment (benblack75) [reply Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material]: I worked with Jonny to get the colors dialed correctly across the board. This involved setting the red that was mistakenly cast as green to its proper value, as well as going into each of the materials and modifying manually to accommodate for source of the color- when material was procedurally based, file texture assignments were removed entirely, and when colors were texture based, basecolor was set to white, texture map was assigned, and texture handling set to lerp (previously set to multiply)

**Review by HogJonny-AMZN — APPROVED** [review]

File: Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material [hunk 2]
```diff
@@ -6,12 +6,11 @@
     "properties": {
         "baseColor": {
             "color": [
-                0.43414968252182007,
-                0.029556725174188614,
-                0.03955138474702835,
+                0.4005493223667145,
```
Comment (Adi-Amazon) [inline Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material:9]: I'm still not quite there yet.  Looking at https://www.babelcolor.com/index_htm_files/RGB%20Coordinates%20of%20the%20Macbeth%20ColorChecker.pdf if you convert the values to [0..1] range (divide by 255) the results are different.

Comment (invertednormal) [reply Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material:9]: I plugged in the numbers for the 16 bit BabelColor average from https://www.babelcolor.com/index_htm_files/RGB%20Coordinates%20of%20the%20Macbeth%20ColorChecker.pdf using our own sRGB->Linear function, and the resulting values are:
0.43244872157
0.0297351510059
0.0399429307193
These should match perfectly with the tif.

Comment (benblack75) [reply Gems/Atom/Feature/Common/Assets/Materials/Presets/MacBeth/15_red.material]: Plugged values- this is the result:
Capture

**Review by invertednormal — APPROVED** [review]

**Pull request merged**